source/neuropod/bindings/java/src/main/native/com_uber_neuropod_LibraryLoader.cc

```diff
@@ -31,3 +32,8 @@ JNIEXPORT void JNICALL Java_com_uber_neuropod_LibraryLoader_nativeExport(JNIEnv
     std::string oriPath = getenv("PATH");
     setenv("PATH", (oriPath + ":" + neuropod::jni::toString(env, libPath)).c_str(), 1 /* Overwrite */);
 }
+
+JNIEXPORT void JNICALL Java_com_uber_neuropod_LibraryLoader_nativeSetTestMode(JNIEnv *, jclass, jboolean mode)
+{
+    isTestMode = mode == JNI_TRUE;
```

Comment:
This is how the test mode is set to true

Reply:
ChangeLog: Add test for TensorSpec symbol case, fix the strange line counting on NeuropodTensor.java, add try catch block for delete functions, merge two nativeNew method for Neuropod into one.